Newsletter2Go/Export/Model/Api/AbstractNewsletter2Go.php

```diff
@@ -49,11 +49,10 @@ public function generateErrorResponse($message)
      */
     public function generateSuccessResponse($data = [])
     {
-        /** @var \Newsletter2Go\Export\Api\Data\ResponseInterface $response */
         $response = $this->responseFactory->create();
         $response->setSuccess(true);
         $response->setMessage('OK');
-        $response->setData($data);
+        $response->setData(json_encode($data));
```

Comment:
This fixes the magento2 problem (by always encoding `$data`), however it depends on [changes in the connector too](https://github.com/newsletter2go/soprex_common/pull/8). @Maiky